document-generation/handlebarUtils/helpers.test.ts

```diff
@@ -0,0 +1,182 @@
+import { HelperOptions } from "handlebars";
+import { AwardType, Classification, ImpactLevel, PeriodType, PeriodUnit } from "../../models/document-generation";
+import { formatDuration, formatGroupAndClassification, formatAwardType, countSections, counter } from "./helpers";
+
+const baseAdnOptionPeriods = [
+  {
+    period_type: PeriodType.OPTION,
+    period_unit_count: 12,
+    period_unit: PeriodUnit.WEEK,
+    option_order: 2,
+  },
+  {
+    period_type: PeriodType.BASE,
+    period_unit_count: 1,
+    period_unit: PeriodUnit.YEAR,
+    // probably good to make bass periods -1 to keep type consistent
```

Comment:
For `base_period` there is no `option_period` so it's left `null`. When `option_period` is `null` it causes some typescript conflicts because of the `number | null` so decided to keep it as only `number` and set the base period `option_period`  to `-1` as an exception.